packages/core/src/app/customer/__snapshots__/CheckoutButtonContainer.spec.tsx.snap

```diff
@@ -65,6 +59,12 @@ exports[`CheckoutButtonContainer displays wallet buttons for guest checkout 1`]
         <div
           id="googlepayauthorizenetCheckoutButton"
         />
+        <div
```

Comment:
It might be better to update CheckoutButtonContainer.spec file and mock availableMethodIds variable in a right order. This snapshot may confuse other developers and it shows wrong information, because paypal related providers returned from BE will be first in the list

Reply:
Good point we should not be editing snapshot files manually.